eras/conway/impl/src/Cardano/Ledger/Conway/Tx.hs

```diff
@@ -82,9 +82,10 @@ getConwayMinFeeTx ::
   Int ->
   Coin
 getConwayMinFeeTx pp tx refScriptsSize =
-  alonzoMinFeeTx pp tx <+> (refScriptsSize <×> refScriptFee)
+  alonzoMinFeeTx pp tx <+> refScriptsFee
   where
-    CoinPerByte refScriptFee = pp ^. ppMinFeeRefScriptCoinsPerByteL
+    refScriptFee = unboundRational (pp ^. ppMinFeeRefScriptCostPerByteL)
+    refScriptsFee = Coin (floor (fromIntegral @Int @Rational refScriptsSize * refScriptFee))
```

Comment:
Let's change the binding name to reflect the meaning a little better
```suggestion
    refScriptCostPerByte = unboundRational (pp ^. ppMinFeeRefScriptCostPerByteL)
    refScriptsFee = 
      Coin (floor (fromIntegral @Int @Rational refScriptsSize * refScriptCostPerByte))
```

Reply:
Understood :pray: